druid/src/env.rs

```diff
@@ -435,10 +438,13 @@ impl Data for Value {
         use Value::*;
         match (self, other) {
             (Point(p1), Point(p2)) => p1.x.same(&p2.x) && p1.y.same(&p2.y),
+            (Size(s1), Size(s2)) => s1.width.same(&s2.width) && s1.height.same(&s2.height),
             (Rect(r1), Rect(r2)) => {
                 r1.x0.same(&r2.x0) && r1.y0.same(&r2.y0) && r1.x1.same(&r2.x1) && r1.y1.same(&r2.y1)
             }
-            (Size(s1), Size(s2)) => s1.width.same(&s2.width) && s1.height.same(&s2.height),
+            (Insets(i1), Insets(i2)) => {
```

Comment:
`Insets` has a `Data` impl lower in the file already (these impls were added after this code was written) so we can just write, `(Insets(i1), Insets(i2)) => i1.same(&i2)`, I think.

Reply:
Oh I missed this, sorry.

Reply:
no worries, I have a followup inbound